packages/strapi-design-system/src/DateTimePicker/DateTimePickerProps.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const DateTimePickerProps = (props) => <div {...props} />;
+
+export const dateTimePickerDefaultProps = {
+  ariaLabel: undefined,
+  clearLabel: 'clear',
+  disabled: false,
+  error: undefined,
+  hint: undefined,
+  label: undefined,
+  labelAction: undefined,
+  onClear: undefined,
+  required: false,
+  size: 'M',
+  step: 1,
+  value: undefined,
+};
+export const dateTimePickerPropTypes = {
+  /**
+   * Aria Label
+   */
+  ariaLabel: PropTypes.string,
+  /**
+   * Clear Label
+   */
+  clearLabel: PropTypes.string,
+  /**
+   * Disabled attribute
+   */
+  disabled: PropTypes.bool,
+  /**
+   * Error message or boolean
+   */
+  error: PropTypes.oneOfType([PropTypes.string, PropTypes.bool]),
+  /**
+   * Hint text
```

Comment:
Where this hint text will be shown

Reply:
it is passed inside the Field wrapper component